apps/dapp-console/app/components/Header/HeaderTab.tsx

```diff
@@ -0,0 +1,50 @@
+import { Route, routes } from '@/app/constants'
+import { cn } from '@/app/lib/utils'
+import Link from 'next/link'
+
+type HeaderTabsProps = {
+  currentRoute: Route['path']
```

Comment:
tiny nit:

```
  currentRoute: string
```

since `currentRoute` is from `usePathname`, it is just a string, and doing Route['path'] doesn't add any restrictions